src/components/TrendingCard/_index.scss

```diff
@@ -0,0 +1,50 @@
+////
+/// Trending card.
+/// @group trending-card
+/// @copyright IBM Security 2019
+////
+
+@import '@carbon/layout/scss/mini-unit';
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/motion/scss/motion';
+@import '@carbon/themes/scss/tokens';
+@import '@carbon/type/scss/styles';
+
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+
+@import '../../globals/motion/index';
+@import '../../platform/body/mixins';
+@import '../Component/mixins';
+
+@include security--component($name: trending-card) {
+  @include carbon--motion($name: standard, $mode: productive);
+
+  display: block;
+  min-width: carbon--mini-units($count: 24);
+  padding: $carbon--spacing-05;
+  background-color: $ui-01;
+  transition-duration: $transition-duration--fast;
+  text-decoration: unset;
+
+  &:focus,
+  &:hover {
+    background-color: $hover-ui;
+  }
+
+  &:focus {
+    @include focus-outline($type: outline);
+  }
+
+  &__title {
+    @include carbon--type-style($name: body-long-01);
+
+    margin-top: 0;
+    margin-bottom: $carbon--spacing-02;
+    color: $body__color__text;
```

Comment:
I'm not familiar with this token... 🤔 Why use it instead of `$text-01`?

Reply:
This is further enforcement of our own rules, and is perhaps overkill - `$body__color__text` refers to `$text-01` under the hood, so in this case, we're saying set the color of the `TrendingCard` text to be that of whatever our body color is.